tests/test_bgp_neighbor_af.rb

```diff
@@ -56,6 +63,67 @@ def clean_af(af_args, ebgp=true)
     [obj_af, dbg]
   end
 
+  # One time setup for XR
+  # this doesn't support the matrix very well but
+  # really only need to test 3 cases (ipv4 uni, l2vpn and a vrf)
+  def setup_ios_xr
+    return if platform == :nexus
+    asn  = 1
+    vrf  = 'aa'
+    nbr  = '1.1.1.1'
+    af   = 'ipv4 unicast'
+    vpn  = 'vpnv4 unicast'
+    evpn = 'l2vpn evpn'
+
+    # Global requirements
+    # route policies
+    str1 = 'route-map-in-name'
+    str2 = 'route-map-out-name'
+    config("route-policy #{str1}", 'end-policy')
+    config("route-policy #{str2}", 'end-policy')
+    config("route-policy #{str1.reverse}", 'end-policy')
+    config("route-policy #{str2.reverse}", 'end-policy')
+    config('route-policy foo_bar', 'end-policy')
+    config('route-policy baz_inga', 'end-policy')
```

Comment:
Would it be reasonable to merge these configs together into a single 'config()' call?